docs/config.rst

```diff
@@ -97,6 +97,10 @@ Optional settings
 
   Expects a list of modules that is used to distinguish among stacktrace frames that belong to your app and ones that are part of libraries or core Elixir.  This is used to better display the significant part of stacktraces.  The logic is greedy, so if your app's root module is ``MyApp`` and your setting is ``[MyApp]``, that module as well as any submodules like ``MyApp.Submodule`` would be considered part of your app.  Defaults to ``[]``.
 
+.. describe:: modules
+
+  Expects a map or a list of `%Mix.Dep{}` to report the loaded dependencies.  To include all Mix dependencies, it can be set to `Mix.Dep.cached()`.
```

Comment:
We probably need to say that it expects the modules to be in the same format as Mix.Dep.cached. I would almost prefer for us to call Mix.Dep.cached() at runtime to this.